flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  description = "Nix-enabled environment for your Android device";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+    home-manager = {
+      url = "github:rycee/home-manager";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, nixpkgs, home-manager, flake-utils }:
+    flake-utils.lib.eachSystem [
+      "aarch64-linux"
+    ]
+      (system:
+        let pkgs = nixpkgs.legacyPackages.${system}; in
+        rec {
+          lib = {
+            nix-on-droid = config: import ./modules {
+              inherit pkgs config;
+              home-manager = (import home-manager { });
+            };
+          };
+
+          apps.nix-on-droid = flake-utils.lib.mkApp {
+            drv = (import ./default.nix { inherit pkgs; });
```

Comment:
Could you call nix-on-droid directly without importing default.nix first? default.nix is only needed for channels, would love to keep this.